File: src/main/java/io/teammion/morefood/item/Item.java
package io.teammion.morefood.item;

import io.teammion.morefood.CreativeTabs;

/**
 * Created by Stefan Wimmer <stefanwimmer128@gmail.com> on 05.06.16.
 */
public class Item extends net.minecraft.item.Item
{
    public Item(String name)
    {
        setUnlocalizedName(name);
        setRegistryName(name);
        
        setCreativeTab(CreativeTabs.FOOD);
    }
}

File: src/main/scala/io/teammion/morefood/item/Item.scala
package io.teammion.morefood.item

import io.teammion.morefood.CreativeTabs
import net.minecraft.item.ItemStack

/**
  * Created by Stefan Wimmer <stefanwimmer128@gmail.com> on 24.07.16.
  */
class Item(name : String) extends net.minecraft.item.Item
{
    setRegistryName(name)
    setUnlocalizedName(name)
    
    setCreativeTab(CreativeTabs.ITEM)
    
    def stack(i : Int) : ItemStack =
        new ItemStack(this, i)
    
    def stack : ItemStack =
        stack(0)
}
